docs/dal/index.rst

```diff
@@ -174,6 +174,31 @@ mean G-band magnitude between 19 - 20:
     2171810342771336704 323.25913736080776  51.94305655940998            19.0
     2180349528028140800  310.5233961869657   50.3486391034819            19.0
 
+To explore more query examples, you can try either the ``description`` 
+attribute for some services. For other services like this one, try 
+the ``examples`` attribute.
+
+.. doctest-remote-data::
+
+    >>> for e in tap_service.examples:
+    ...     print(e['QUERY'])   # doctest: +SKIP
+    <query examples>
```

Comment:
Thanks for the review. 

Actually, all the ``doctest: +SKIP`` I put in the doc are because of their output EITHER are temporarily i.e. might be different later in time, OR the outputs are simply too long (that the output can not be shown fully), as far as I understand from ``doctest``, it checks the code output 1 to 1 with the strings that I wrote in the doc.
In this case, since the output is related to the names of the tables, it can indeed be updated later in time, and the list of the names is also too long. I am not sure if there is an ambiguous way for ``doctest`` to solve this issue.  I might go back to doctest documentation and check further details.

And for the variable issues, thanks for that hint, sure I will fix it later asap.

Reply:
Ah also here, the output for the query examples are also too long, there are many examples shown in the output, I could try too write them 1 to 1 into the doc, if that is okay.

Reply:
for long outputs ellipses would work, or you can rewrite the example of looking for one special case in there.

For variable outputs, `doctest: +IGNORE_OUTPUT` is way more preferable than skipping it altogether, as it at least runs the line and spots any syntax, etc issues even if it doesn't compare the expected and actual outputs.

Reply:
Thanks for the hints. I have changed the `doctest` flags accordingly. Also I have to leave one `docktest: +SKIP` to line 319: `job.fetch_result()`. This is because the actual query is not faster than `doctest`'s processing speed, so it would turn out an error that the result does not exist yet.
Furthermore I have optimised some code outputs so that `docktest: +ELLIPSIS` would not be needed.
And  `dockets: + IGNORE_OUTPUTS` is indeed used for variable outputs.